database-enclave/.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
+../.gradle/
+.idea
+**/build/
+**/bin/
+*/.gradle/
+*/.idea/*
+**/.vscode/*
+**/out/
+**/logs/
```

Comment:
Github is telling me that we miss many "newlines" at the end of files, can you please just add them?